### PR TITLE
[KK-0005]: :lipstick: Modified README.md file

### PR DESCRIPTION
**Pull request by kinga-kus**
In this PR, I have modified a view of README.md file. I have added important information and make it look cleaner.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,72 @@
-# Getting Started with Create React App
+## Blood Donation <img width="10%" src="./src/assets/logo.png" alt="Logo" />
```
Comment (adam-kostuch) [inline README.md:1]: I think that it would look better with the smaller logo. It kinda looks strange here 😃

**Review by adam-kostuch — CHANGES_REQUESTED** [review]
Although the file itself looks good, please remember to add some screenshots here once the project gets bigger.

And please before merging remove the files that doesn't belong here ❌

File: README.md (hunk 1 above)
Comment (kinga-kus) [reply README.md:1]: Yes, in the nearest future I'll add some screenshoots. The smaller logo means it should be for example 5% ?

Comment (adam-kostuch) [reply README.md:1]: Can't tell you what it looks like in 5%, but that would probably fit much better for sure!

**Review by adam-kostuch — CHANGES_REQUESTED** [review]
The README is looking 🔥 

But there are still 4 more unnecessary files changed in this PR, please delete them before merging. See this [thread](https://stackoverflow.com/a/44355701) for help.

**Review by adam-kostuch — APPROVED** [review]
Well done!!!! 🔥

**Pull request merged**